### PR TITLE
add recapture & compare and some other doc

### PR DESCRIPTION
**Pull request by yongjuntian**

### CODE REVIEW

File: doc/FAQ.md [hunk 1]
```diff
@@ -9,4 +9,37 @@ sidebar_label: FAQ
...
-If this setting is checked, third-party scripts cookies are disallowed and access to localStorage may result in thrown SecurityError exceptions.
\ No newline at end of file
+If this setting is checked, third-party scripts cookies are disallowed and access to localStorage may result in thrown SecurityError exceptions.
+
+## Can not automate browser running instances with different user profiles
+If you encountered the subsequent issue when recording:
+
+![multiple user profile error](img/faq_multiple_profile_error.png)
+
+This is due to the possibility that you have started several Chrome browser windows using several user profiles. Every Chrome browser window's user profile is visible in the top-right corner.
```
Comment (kayYOLO) [inline doc/FAQ.md:19]: using different user profiles

File: doc/FAQ.md [hunk 2]
```diff
@@ -9,4 +9,37 @@ sidebar_label: FAQ
...
+
+![multiple user profile](img/faq_mulitple_profile2.png)
+
+You can fix this issue by first stopping Clicknium Recorder, closing all open browser tabs, opening the browser solely in a single user profile, and then starting a new recording.
+
+## This element is not similar to previous elements, please capture again
```
Comment (kayYOLO) [inline doc/FAQ.md:25]: This element is not similar to previous elements. Please capture it again

File: doc/FAQ.md [hunk 3]
```diff
@@ -9,4 +9,37 @@ sidebar_label: FAQ
...
+
+![similar element error](img/faq_similar_elelemt_error.png)
+
+The new recorded locator being different from the old one should be the primary factor. For example:
+- They are elements of various types, for example: one with tag "a" and the other with tag "div".
+- They are under different iframes.
+
+For more information, please refer to [Capture similar elements](tutorial/recorder/capture_similar_elements.md)
+
+## Browser extension for Chrome is not turned on, do you want to turn on it now?
```
Comment (kayYOLO) [inline doc/FAQ.md:36]: turn it on

File: doc/FAQ.md [hunk 4]
```diff
@@ -9,4 +9,37 @@ sidebar_label: FAQ
...
+If you encountered the subsequent issue when recording element on one browser page,
```
Comment (kayYOLO) [inline doc/FAQ.md:37]: recording elements

File: doc/how-to/how-to-use-web-custom-attributes.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+[Clicknium Python Module](https://pypi.org/project/clicknium/) >= 0.1.8
+
+:::tip Notes
+
+More about the installation and the tutorial of Clicknium Automation, please refer to [here](https://www.clicknium.com/documents).
+
+:::
+## Samples
+The example below demonstrates how to make advantage of this functionality.
```
Comment (kayYOLO) [inline doc/how-to/how-to-use-web-custom-attributes.md:21]: take advantage

File: doc/tutorial/recorder/recapture&compare.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+# Recapture & Compare
+
+## Version Requirement
+[Clicknium Visual Studio Code Extension](https://marketplace.visualstudio.com/items?itemName=ClickCorp.clicknium) >= 0.1.9
+
+[Clicknium Python Module](https://pypi.org/project/clicknium/) >= 0.1.8
+
+## Overview
+
+Recapture & Compare allows you to recode the same UI element, create a new location, and present the user with a comparable view.
```
Comment (kayYOLO) [inline doc/tutorial/recorder/recapture&compare.md:14]: recode the same UI element? or record?

File: doc/tutorial/recorder/recapture&compare.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+The user may easily distinguish the dynamic attributes by comparing the new locator to the prior one. The chosen list should not include these dynamic attributes.
+
+Many websites today include dynamic pages, which means that when a user opens the same page, different material is shown. Locating the element will be challenging in this scenario. The element's locator was successfully located the previous time, and it might be failed located the next time. . Use the "Recapture and Compare" feature to rapidly decide which properties on the locator should be used in this situation.
```
Comment (kayYOLO) [inline doc/tutorial/recorder/recapture&compare.md:17]: extra  .. you can update this doc using the branch I refactored yesterday.

File: doc/tutorial/recorder/recapture&compare.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+The example below demonstrates how to make advantage of this functionality.
```
Comment (kayYOLO) [inline doc/tutorial/recorder/recapture&compare.md:19]: take advantage of

File: doc/how-to/how-to-use-variable-in-locator-find-elements.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+
+:::
+
+## Scraping contact list on slack client
+
+This is a real user scenario, which needs to grab user list information from a channel of slack. 
+
+![slack contact list](img/slack_contact_list.png)
+
+After record one contact item's locator, user modify the locator with index variable.
```
Comment (DiceyClickcorp) [inline doc/how-to/how-to-use-variable-in-locator-find-elements.md:24]: recording

**Review by DiceyClickcorp — APPROVED** [review]

**Pull request merged**